includes/definitions/discovery/helios.yaml

```diff
@@ -0,0 +1,12 @@
+modules:
+    sensors:
+        state:
+            data:
+                -
+                    oid: IGNITENET-MIB::mlRadioStatusFailoverStatus.2
+                    value: mlRadioStatusFailoverStatus
+                    num_oid: .1.3.6.1.4.1.47307.1.4.3.1.2.2
+                    descr: 60GHz to 5GHz Radio Failover Status
+                    states:
+                        - { value : 0, generic : 0, graph : 0, descr : Failover Inactive }
```

Comment:
Maybe better descriptions would be:
"Failover Status"
"60GHz"
"5GHz"

I think that would be a lot cleaner.  And you get the yellow color to indiate 5GHz is the failover status.